File: Day-09 ( BMI Category )/Python/Solution.py
weight = float(input("Enter your weight in kg: "))
height = float(input("Enter your height in meters: "))

bmi = weight / (height * height)

print(f"BMI: {bmi:.2f}")

if bmi < 18.5:
    print("Category: Underweight")
else:
    if bmi < 25:
        print("Category: Normal weight")
    else:
        if bmi < 30:
            print("Category: Overweight")
        else:
            print("Category: Obese")

File: Day-09 ( BMI Category )/CPP/Solution.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {
    double weight, height;

    cout << "Enter your weight in kg: ";
    cin >> weight;

    cout << "Enter your height in meters: ";
    cin >> height;

    double bmi = weight / (height * height);

    cout << fixed << setprecision(2);
    cout << "BMI: " << bmi << endl;

    if (bmi < 18.5) {
        cout << "Category: Underweight" << endl;
    }
    else {
        if (bmi < 25) {
            cout << "Category: Normal weight" << endl;
        }
        else {
            if (bmi < 30) {
                cout << "Category: Overweight" << endl;
            }
            else {
                cout << "Category: Obese" << endl;
            }
        }
    }

    return 0;
}

